app/models/module_item.rb

```diff
@@ -21,6 +21,7 @@ def self.load_file
   MODELS = {
     assessment_intro: ContentPage,
     confidence_intro: ContentPage,
+    ending_intro: ContentPage,
```

Comment:
I think we should now split these so we can `#comment` that "intros" and "pages" are sorted in two groups